os/hal/hal.mk

```diff
@@ -95,6 +95,11 @@ endif
 ifneq ($(findstring HAL_USE_WSPI TRUE,$(HALCONF)),)
 HALSRC += $(CHIBIOS)/os/hal/src/hal_wspi.c
 endif
+ifneq ($(findstring HAL_USE_USB_MSD TRUE,$(HALCONF)),)
```

Comment:
Are you sure this works? I was having problems yesterday

Reply:
Its working fine last I tested